wavenet.py

```diff
@@ -51,16 +52,33 @@ def _create_dilation_layer(self, input_batch, layer_index, dilation, in_channels
         conv_filter = causal_conv(input_batch, weights_filter, dilation)
         conv_gate = causal_conv(input_batch, weights_gate, dilation)
 
+        if self.use_biases:
+          biases_filter = tf.Variable(tf.constant(0.0, shape=[dilation_channels]),
```

Comment:
As 4-spaces is used for other parts, I suggest to use it instead of 2-spaces for indentation.